pandas/tests/indexes/timedeltas/test_timedelta.py

```diff
@@ -401,3 +401,44 @@ def test_series_box_timedelta(self):
         s = Series(rng)
         assert isinstance(s[1], Timedelta)
         assert isinstance(s.iat[2], Timedelta)
+
+
```

Comment:
make a note to parametrize this

Reply:
Not clear what "this" is in context.

Reply:
this is the calling of testit, simple enough should be to paramerize the units which its iterating